docs/assembly.rst

```diff
@@ -596,43 +596,43 @@ of their block is reached.
 Conventions in Solidity
 -----------------------
 
-In contrast to EVM assembly, Solidity knows types which are narrower than 256 bits,
-e.g. ``uint24``. In order to make them more efficient, most arithmetic operations just
-treat them as 256-bit numbers and the higher-order bits are only cleaned at the
-point where it is necessary, i.e. just shortly before they are written to memory
-or before comparisons are performed. This means that if you access such a variable
-from within inline assembly, you might have to manually clean the higher order bits
+In contrast to EVM assembly, Solidity knows which types are narrower than 256 bits,
```

Comment:
This was meant to say that Solidity has such types at all, not that it knows which are which.